libs/net/dal/Services/ReportService.cs

```diff
@@ -829,8 +828,14 @@ public IEnumerable<ReportInstance> GetLatestInstances(int id, int? ownerId = nul
                 if (sectionSettings.RemoveDuplicates)
                     query = query.Where(fc => !excludeAboveSectionContentIds.Contains(fc.ContentId));
 
-                if (excludeContentIds.Any() && !sectionSettings.OverrideExcludeHistorical)
-                    query = query.Where(fc => !excludeContentIds.Contains(fc.ContentId));
+                if (!sectionSettings.OverrideExcludeHistorical)
```

Comment:
Fixed logic for excluding content from a linked report.

Fixed logic for excluding content from above sections.